README.md

```diff
@@ -206,7 +206,7 @@ selections to it. This works well for evaluating things in GHCI.
     <td>,tt</td><td>Open/close the tag bar</td>
   </tr>
   <tr>
-    <td>C-]</td><td>Jump to definition of symbol (codex + hasktags)</td>
+    <td>C-]</td><td>Jump to definition of symbol (codex + hasktags)</td><td>Note: You must generate the tags for your project (using `,tg`) prior to using the jump command.</td> 
```

Comment:
Cool, I didn't know you could do a variable number of td's in a table but it looks good in preview. One thing is you should use `<code>,tg</code>` here rather than backticks because the github markdown renderer is in html mode at this point.